lib/rules/no-duplicate-at-import-rules/README.md

```diff
@@ -0,0 +1,48 @@
+# no-duplicate-at-import-rules
+
+Disallow duplicate `@import` rules within a stylesheet.
+
+```css
```

Comment:
It's more common to see `@import`s on single lines, so let's have the example like that:

```css
    @import "a.css";
    @import "a.css";
/** ↑
* These duplicates  */
```